payload-builder/src/service.rs

```diff
@@ -71,6 +71,9 @@ impl<L: Ledger> PayloadService<L> {
     }
 
     fn print_statisitc(&self) {
+        if self.tasks.is_empty() {
+            return;
+        }
```

Comment:
This is never the case for `op-sol` right?

Reply:
This is a rare case in production, but mainly for use in local work.